scripts/PortalFilesTester.py

```diff
@@ -7,6 +7,7 @@
 
```

Comment:
Please rename this to `portal_files_tester.py` via `git mv PortalFilesTester.py portal_files_tester.py`.  See other comment for rationale.

Reply:
This can be deferred.